units/Legion/Economy/legmohoconin.lua

```diff
@@ -0,0 +1,81 @@
+return {						--costs should be same as legmohocon and legmohoconct
+	legmohoconin = {
+		maxacc = 0,
+		activatewhenbuilt = true,
+		maxdec = 0,
+		buildangle = 2048,
+		energycost = 8100,
+		metalcost = 640,
+		buildingmask = 0,
+		buildpic = "LEGMOHOCON.DDS",
+		buildtime = 14100,
+		canrepeat = false,
+		collisionvolumeoffsets = "0 -1 0",
+		collisionvolumescales = "7 4 7",
+		collisionvolumetype = "CylY",
+		corpse = "DEAD",
+		energyupkeep = 20,
+		explodeas = "largeBuildingexplosiongeneric",
```

Comment:
should the hidden one have an explosion?

Reply:
It would never explode, but removed it anyway.